### PR TITLE
enable chain config banner for testnet

### LINKED ISSUE #71: Display chain banner for testnet
When started for Testnet app doesn't show information banner with forks/version details

### PR DESCRIPTION
**Pull request by splix**
fix #71

example:
```
I1018 17:14:39.314252 cmd/utils/flags.go:824] --------------------------------------------------------------------------------------------------------------
I1018 17:14:39.314280 cmd/utils/flags.go:825] Starting Geth Classic 3.0.0-stable-b552d46c
I1018 17:14:39.314339 cmd/utils/flags.go:826] Loading blockchain: genesis block 0x0cd786a2425d16f152c658316c423e6ce1181e15c3295826d7c9904cba9ce303.
I1018 17:14:39.314352 cmd/utils/flags.go:827] 4 blockchain hard-forks associated with this genesis block:
I1018 17:14:39.314377 cmd/utils/flags.go:839]  Homestead hard-fork at block 494000
I1018 17:14:39.314400 cmd/utils/flags.go:839]  GasReprice hard-fork at block 1783000
I1018 17:14:39.314413 cmd/utils/flags.go:843] Geth is configured to use the Ethereum (ETC) Testnet blockchain!
I1018 17:14:39.314421 cmd/utils/flags.go:847] --------------------------------------------------------------------------------------------------------------
```

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -819,33 +820,33 @@ func MustMakeChainConfigFromDb(ctx *cli.Context, db ethdb.Database) *core.ChainC
...
-				netsplitChoice = ""
-			}
+	separator := strings.Repeat("-", 110)
+	glog.V(logger.Warn).Info(separator)
+	glog.V(logger.Warn).Info(fmt.Sprintf("Starting Geth Classic \x1b[32m%s\x1b[39m", ctx.App.Version))
+	glog.V(logger.Warn).Info(fmt.Sprintf("Loading blockchain: \x1b[36mgenesis\x1b[39m block \x1b[36m%s\x1b[39m.", genesis.Hash().Hex()))
+	glog.V(logger.Warn).Info(fmt.Sprintf("%v blockchain hard-forks associated with this genesis block:", len(c.Forks)))
+
+	netsplitChoice := ""
+	testnet := []string{"Homestead", "GasReprice"} //TODO create separate chain config for testnet, then remove that
```
Comment (elaineo) [inline cmd/utils/flags.go]: Does "DieHard" need to be added, or does it automatically become enabled if GasReprice is enabled?

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -819,33 +820,33 @@ func MustMakeChainConfigFromDb(ctx *cli.Context, db ethdb.Database) *core.ChainC
...
+	sort.Strings(testnet)
+
+	for i := range c.Forks {
+		if c.Forks[i].NetworkSplit {
+			netsplitChoice = fmt.Sprintf("resulted in a network split (support: %t)", c.Forks[i].Support)
+		} else {
+			netsplitChoice = ""
+		}
+		testnetEnabled := sort.SearchStrings(testnet, c.Forks[i].Name) < len(testnet) && testnet[sort.SearchStrings(testnet, c.Forks[i].Name)] == c.Forks[i].Name
```
Comment (elaineo) [inline cmd/utils/flags.go]: is this the best way to determine if the fork exists in the list? (not familiar enough with Go, it just seems like a roundabout way to do it)

File: cmd/utils/flags.go (hunk 1 above)
Comment (splix) [inline cmd/utils/flags.go]: It's a quick hack for current testnet (notice TODO comment), we'll make it better in following commits

File: cmd/utils/flags.go (hunk 2 above)
Comment (splix) [inline cmd/utils/flags.go]: Yes, it looks crazy, agree, but I didn't find better way. StackOverflow suggests that

**Pull request merged**